agent/01_agent_install_requirements.sh

```diff
@@ -0,0 +1,28 @@
+# #!/usr/bin/env bash
+set -euxo pipefail
+
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-$(systemd-path user-configuration)}
+SCRIPTDIR="$(pwd)"
+USER=`whoami`
+
+# Get variables from the config file
+if [ -z "${CONFIG:-}" ]; then
+    # See if there's a config_$USER.sh in the SCRIPTDIR
+    if [ -f ${SCRIPTDIR}/config_${USER}.sh ]; then
+        echo "Using CONFIG ${SCRIPTDIR}/config_${USER}.sh" 1>&2
+        CONFIG="${SCRIPTDIR}/config_${USER}.sh"
+    elif [[ -f "${XDG_CONFIG_HOME}/dev-scripts/config" ]]; then
+        echo "Using CONFIG ${XDG_CONFIG_HOME}/dev-scripts/config" 1>&2
+        CONFIG="${XDG_CONFIG_HOME}/dev-scripts/config"
+    else
+        error "Please run with a configuration environment set."
+        error "eg CONFIG=config_example.sh ./01_all_in_one.sh"
+        exit 1
+    fi
+fi
+source $CONFIG
```

Comment:
A similar code block exist in common.sh. I think we can either turn it into a function and reuse the code block that way.

Or remove these lines entirely. Update the Makefile with requirement listed ahead of agent_requirements in the "agent" target. And removing requirements from the agent_requirements target.:
```

agent: requirements agent_requirements configure agent_build_installer agent_configure agent_create_cluster

agent_requirements:
	./agent/01_agent_install_requirements.sh
```

Reply:
I guess that if we keep `AGENT_CONFIG` in common.sh, evaluated only when defined (see comments below), we could avoid at all this part